spec/services/notifications/welcome_notification/send_spec.rb

```diff
@@ -2,18 +2,31 @@
 
 RSpec.describe Notifications::WelcomeNotification::Send, type: :service do
   describe "::call" do
+    let!(:welcome_broadcast) { create(:set_up_profile_broadcast) }
+
     before do
       allow(User).to receive(:mascot_account).and_return(create(:user))
+      allow(DatadogStatsClient).to receive(:increment)
     end
 
     it "creates a new welcome notification", :aggregate_failures do
-      welcome_broadcast = create(:set_up_profile_broadcast)
-      welcome_notification = described_class.call(create(:user).id, welcome_broadcast)
```

Comment:
I'm not sure how this was ever working...it just kept failing for me locally, and didn't appear to be returning the correct object, even. 🤔 I went ahead and refactored this file while adding in my spec!